packages/rs-platform-version/src/version/fee/data_contract/mod.rs

```diff
@@ -10,3 +10,18 @@ pub struct FeeDataContractValidationVersion {
     pub document_type_base_unique_index_fee: u64,
     pub document_type_unique_index_per_property_fee: u64,
 }
+
+impl PartialEq for FeeDataContractValidationVersion {
+    fn eq(&self, other: &Self) -> bool {
```

Comment:
If you do like this better you have a test to make sure this thing won't break when you add more fields in future.

Reply:
Does FeeVersion has an integer representation?

Reply:
Implemented `to_hash()` for `FeeVersion` and all of its fields. But I feel this is over engineering

Reply:
So that we can compare two `FeeVersion`

Reply:
May point was to add a test where we would create this struct `FeeVersion { .. }` so it will fail when we add new fields to this structure and when you fix it you will see that you need to update these comparison functions